src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -425,18 +424,10 @@ private async void RecycleBinItemDeleted(object sender, FileSystemEventArgs e)
 			if (!Constants.UserEnvironmentPaths.RecycleBinPath.Equals(CurrentFolder?.ItemPath, StringComparison.OrdinalIgnoreCase))
 				return;
 
-			// Get the item that immediately follows matching item to be removed
-			// If the matching item is the last item, try to get the previous item; otherwise, null
-			// Case must be ignored since $Recycle.Bin != $RECYCLE.BIN
-			var itemRemovedIndex = filesAndFolders.FindIndex(x => x.ItemPath.Equals(e.FullPath, StringComparison.OrdinalIgnoreCase));
-			var nextOfMatchingItem = filesAndFolders.ElementAtOrDefault(itemRemovedIndex + 1 < filesAndFolders.Count ? itemRemovedIndex + 1 : itemRemovedIndex - 1);
```

Comment:
I don't think this is related

Reply:
An implementation of `FindIndex()` is needed to keep this process. However, this process is not necessary here either, since it is used to select a neighboring file upon deletion, a process that has already been deleted except in the recycle bin.